tests/cypress/integration/notifications.cy.js

```diff
@@ -0,0 +1,60 @@
+// <reference types="Cypress" />
+const marketplace = require('../fixtures/notifications.json');
+
+describe('Notifications', function () {
+
+	before(() => {
```

Comment:
@circlecube The `before` function only runs once for all the tests. However, [intercepts are cleared before every test](https://docs.cypress.io/api/commands/intercept#Syntax:~:text=All%20intercepts%20are%20automatically%20cleared%20before%20every%20test.). As a result, you should move the intercept portion to a `beforeEach` (and leave the rest as is).